custom_components/frigate/api.py

```diff
@@ -47,7 +47,7 @@ async def async_get_version(self) -> str:
     async def async_get_stats(self) -> dict[str, Any]:
         """Get data from the API."""
         return cast(
-            Dict[str, Any],
+            dict[str, Any],
```

Comment:
Just curious, what's the reason for using the built in type? In Frigate upstream it was switched the other way, not sure if there's a real distinction between `Dict` and `dict`

Reply:
The old type hints (with the capital letters) are now retired (https://peps.python.org/pep-0585/#implementation) since Python ~3.9 IIRC. It used to be that the built-in types could be used everywhere except casts (like this), but that too seems to have been fixed now, hence this change to make this code more future proof.